.travis.d/templates/deployment.yaml.erb

```diff
@@ -34,10 +46,16 @@ spec:
                 value: !!str <%= app['target_port'] or 3000 %>
               - name: ROOT_URL
                 value: <%= "https://#{app['host']}" %>
-              <% if app['wants'] && app['wants']['mongo'] %>
-              - name: MONGO_URL
+              <% if app['wants']%>
+              <% if app['wants']['mongo'] %>
+              - name: <%= app['wants']['mongo'].is_a?(String) ? app['wants']['mongo'] : 'MONGO_URL' %>
```

Comment:
Could you do:
```
wants:
  mongo:
    env: MONGO_URL
```
instead, I feel like in the future we might want to add more options here.
Note that an empty object and `true` in Ruby are both truthy, so this shouldn't effect so much stuff.